### PR TITLE
[Feature] Sentry Logs

### PR DESCRIPTION
**Pull request by Swiizyy**
### Feats
- Add dependencies
- Add Sentry Implementation

Closes: #107

### CODE REVIEW

File: src/lib/BirthdayyClient.ts [hunk 1]
```diff
@@ -16,25 +16,4 @@ export class BirthdayyClient extends SapphireClient {
 	private registerPaths() {
 		this.stores.get('scheduled-tasks').registerPath(join(getRootData().root, 'tasks'));
 	}
-
-	public async login(token?: string): Promise<string> {
```
Comment (nikolaischunk) [inline src/lib/BirthdayyClient.ts:20]: why have you deleted the login?

File: src/index.ts [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+import { SENTRY_OPTIONS } from './config';
+import { SENTRY_DSN } from './helpers/provide/environment';
 
 const client = new BirthdayyClient();
 
-client.login();
+async function main() {
+	try {
+		// Initialize Sentry
+		SENTRY_DSN ?? Sentry.init(SENTRY_OPTIONS);
```
Comment (nikolaischunk) [inline src/index.ts:14]: this statement makes no sense or am I missing something.
grafik
the init gets called with the options when `SENTRY_DSN` is nullish.
But in the options, you define the `dsn` as exactly the `SENTRY_DSN`. 

So you say: `if SENTRY_DSN is nullish execute init with the sentry options that have the dsn included?`

Comment (Swiizyy) [reply src/index.ts:14]: check => https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Comment (Swiizyy) [reply src/index.ts:14]: oh my bad..

Comment (Swiizyy) [reply src/index.ts]: Oh my bad...

**Pull request merged**